### ISSUE TITLE
Text of the icon view in lxqt-config-session has wrong width, making it unreadable

### ISSUE THREAD

**Issue by PCMan** [lxqt/lxqt#1510]
Text of the icon view in lxqt-config-session has wrong width, making it unreadable

##### Expected Behavior
Text should be wider than the icon and only wrap when absolutely needed.

##### Current Behavior
If you use a non-English locale, like traditional Chinese (zh_TW), it becomes like this (one character per line).
![lxqt-config-session](https://user-images.githubusercontent.com/46543/42124221-6af371e2-7c91-11e8-8c2e-334f76183cbc.png)

##### Possible Solution
Fix the icon view UI element in liblxqt to allow wider text.

##### Steps to Reproduce (for bugs)
1. > export LC_ALL=zh_TW.UTF-8
2. > lxqt-config-session

##### System Information
* Distribution & Version: ArchLinux
* Kernel: Linux arch-pc 4.17.2-1-ARCH #1 SMP PREEMPT Sat Jun 16 11:08:59 UTC 2018 x86_64 GNU/Linux
* Qt Version: 5.11.1
* liblxqt Version: 0.13.0
* Package version:

**Comment by PCMan** [comment]
@yan12125 do you encounter this issue?

**Comment by yan12125** [comment]
> @yan12125 do you encounter this issue?

Yes, and it's not limited to lxqt-config-session. For example, lxqt-config-appearance,
![default](https://user-images.githubusercontent.com/1937689/42124942-43088ff6-7c9f-11e8-96b4-9426510b6040.png)
and the "configure panel" dialog of lxqt-panel,
![default](https://user-images.githubusercontent.com/1937689/42124968-c6470f6e-7c9f-11e8-89ae-62c542eda143.png)

**Comment by agaida** [comment]
@tsujan - portugese is fine in some parts
![](https://i.imgur.com/1xvkjoO.png)

zh_HK
![](https://i.imgur.com/2q3qDD4.png)

zh_CN
![](https://i.imgur.com/1HNGbvt.jpg)

so i would say better, but still a way to go - esp the lists.

**Comment by tsujan** [comment]
If yes, tell me what languages they are for me to find the cause by testing later.

**Comment by tsujan** [comment]
OK, saw them in your terminal screenshots -- these screenshots are VERY big ;)

**Comment by tsujan** [comment]
Damn! The same here:

![bad](https://user-images.githubusercontent.com/981076/48105164-a72d7980-e24b-11e8-9a1a-d10af598e20a.png)

**Comment by tsujan** [comment]
I can make it not so terrible by a simple change:

![not_bad](https://user-images.githubusercontent.com/981076/48105301-23c05800-e24c-11e8-96dc-00829a01e6d7.png)

But that isn't good yet: items are too tall.

So, please be patient, don't curse me, and let me investigate the cause ;)

**Comment by tsujan** [comment]
@agaida Please test https://github.com/lxqt/liblxqt/pull/187 with as many languages as you can and with your huge monitor. It seems fixed here but I can't be sure:

![fixed](https://user-images.githubusercontent.com/981076/48108048-ec0add80-e256-11e8-98ae-f6e256486856.png)

I guess you're asleep and dreaming. It's my turn now. Till 9 hours later!

**Comment by agaida** [comment]
It's now 13h later - here it is 17:16, just back from work - so let's start and mess up things :D

**Comment by agaida** [comment]
Great!! GTM.
![image](https://user-images.githubusercontent.com/1162845/48151028-86ebd200-e2c0-11e8-9c3c-2132a06adc37.png)

And i couldn't resist, but find the bug in RTL - should be handled in a separated bug.

**Comment by tsujan** [comment]
Wow! What a comprehensive test! Very good. Thanks!

So, I'm not a bad coder when I'm drowsy ;)

> And i couldn't resist, but find the bug in RTL

I think I see it in your screenshot but it isn't related to this problem.

I'll also make another PR for another bug in liblxqt.

**Comment by tsujan** [comment]
Looks OK.

BTW, if by RTL problem you mean the cut texts and widgets, it's a very old Qt issue that has never been fixed -- it was present even in Qt4. But some widget style show it more than others.

**Comment by tsujan** [comment]
And I hope the widget style you used in your screenshot doesn't have any relation to Kvantum because its RTL is buggy.

**Comment by agaida** [comment]
it's Adwaita - afaik
![grafik](https://user-images.githubusercontent.com/1162845/48152475-4e4df780-e2c4-11e8-91de-ed8fa380817d.png)

**Comment by agaida** [comment]
ok

**Comment by tsujan** [comment]
RTL can be tested with any language by this command: `APP --reverse` -- `lxqt-config-appearance --reverse`, for example.

![reverse](https://user-images.githubusercontent.com/981076/48152956-64ff4900-e2da-11e8-9e33-591f97455815.png)
